src/app/forms/custom-sections/custom-defects/custom-defects.component.html

```diff
@@ -0,0 +1,24 @@
+<ng-container *ngIf="form">
+  <ng-container [formGroup]="form">
+    <ng-container *ngIf="defectCount; else noDefects">
+      <h2 class="govuk-heading-m">Custom Defects</h2>
+
+      <app-custom-defect
+        *ngFor="let customDefect of customDefects; let i = index; trackBy: trackByFn"
+        [isEditing]="isEditing"
+        [form]="getCustomDefectForm(i)"
+        [index]="i"
+        (removeCustomDefect)="handleRemoveDefect($event)"
+      ></app-custom-defect>
+    </ng-container>
+  </ng-container>
+</ng-container>
```

Comment:
You can remove the nesting here
```suggestion
<ng-container *ngIf="form && defectCount; else noDefects" [formGroup]="form">
  <h2 class="govuk-heading-m">Custom Defects</h2>

  <app-custom-defect
    *ngFor="let customDefect of customDefects; let i = index; trackBy: trackByFn"
    [isEditing]="isEditing"
    [form]="getCustomDefectForm(i)"
    [index]="i"
    (removeCustomDefect)="handleRemoveDefect($event)"
  ></app-custom-defect>
</ng-container>
```